### PR TITLE
Add SECURITY.md to outline security issue reporting

### PR DESCRIPTION
**Pull request by lazyguru**
Add a SECURITY.md file to provide guidelines for reporting security issues. This will help in the responsible disclosure of security bugs by detailing the process for reporting vulnerabilities. It includes a link to the GitHub Security Advisory for ease of access and ensures contributors are informed about the handling of their reports.

### CODE REVIEW

**Review by kgilles — APPROVED** [review]

**Review by kgilles — CHANGES_REQUESTED** [review]
Code owners file contains errors. Not sure the exact format it wants, but looks like the teams need to be updated to something like `@sublinks/developers/sublinks-frontend-service`

**Comment by lazyguru** [comment]
Hmm... what's broken on it? Pretty sure this: @sublinks/sublinks-frontend-service works (same format that is in the file and I didn't change those)

**Comment by kgilles** [comment]
@lazyguru Are you seeing this if you go to review the file changes?
image

**Comment by lazyguru** [comment]
In the app I don't: image

**Comment by lazyguru** [comment]
On the web I do though:
image

**Comment by lazyguru** [comment]
Found the problem:
image

It's the part about write access.

**Comment by kgilles** [comment]
> Found the problem: image
> 
> It's the part about write access.

Great! I just found the small dots that opened the detailed explanation 😓

**Comment by lazyguru** [comment]
I guess it kind of makes sense that the group would need write access, but the idea was that I want their approval but also want to control which PRs get merged. I'll have to think on it, but for now the main change is renaming developer to owner

**Review by kgilles — APPROVED** [review]

**Pull request merged**
